src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -160,12 +160,14 @@ public DynamoAnalyticsClient(DynamoModel dynamoModel)
         /// </summary>
         public void Start()
         {
-            //If not ReportingAnalytics, then set the idle time as infinite so idle state is not recorded.
-            Service.StartUp(product,
-                new UserInfo(Session.UserId), ReportingAnalytics ? TimeSpan.FromMinutes(30) : TimeSpan.MaxValue);
-
-            TrackPreferenceInternal("ReportingAnalytics", "", ReportingAnalytics ? 1 : 0);
-            TrackPreferenceInternal("ReportingUsage", "", ReportingUsage ? 1 : 0);
+            if (ReportingAnalytics || ReportingUsage)
```

Comment:
shouldn't this be `&&` now?

for those users who still have old preference settings xml files where they might have instrumentation on but analytics off - I think this will turn on instrumentation, but make it hard or impossible for them to turn it off without first turning analytics off.

Reply:
Yeah I made this change before our SPIKE, I will use only `ReportingAnalytics` now since that should dominate the other one in our GUI